packages/core/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,169 @@
+import FormLabel from '../FormLabel/FormLabel';
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash.uniqueid';
+
+/**
+ * A `Dropdown` component can be used to render an HTML `select` menu.
+ * Any _undocumented_ props that you pass to this component will be passed
+ * to the `select` element, so you can use this to set additional attributes if
+ * necessary.
+ *
+ * Class-based component gives flexibility for active focus management
+ * by allowing refs to be passed.
+ */
+export class Dropdown extends React.PureComponent {
+  componentDidMount() {
+    if (this.props.focusTrigger) {
+      this.loader && this.loader.focus();
+    }
+  }
+
+  id() {
+    if (!this._id) {
+      // Cache the ID so we're not regenerating it on each method call
+      this._id = uniqueId(`select_${this.props.name}_`);
+    }
+    return this._id;
+  }
+
+  render() {
+    /* eslint-disable prefer-const */
+    let {
+      // Using let rather than const since we sometimes rewrite id
```

Comment:
Looks like this doesn't apply anymore. I don't see `id` in this list or any of these variables being mutated.